src/components/Canvas/Canvas.wrapper.tsx

```diff
@@ -19,8 +19,8 @@ export interface ICanvasWrapperProps {
   onDeleteKey: IOnDeleteKey
   onCanvasClick: IOnCanvasClick
   onCanvasDrop: IOnCanvasDrop
-  ComponentInner: React.FunctionComponent<ICanvasInnerDefaultProps>
-  ComponentOuter: React.FunctionComponent<ICanvasOuterDefaultProps>
+  ComponentInner?: React.FunctionComponent<ICanvasInnerDefaultProps>
+  ComponentOuter?: React.FunctionComponent<ICanvasOuterDefaultProps>
```

Comment:
Ze mogen niet undefined zijn.
```suggestion
  ComponentInner: React.FunctionComponent<ICanvasInnerDefaultProps>
  ComponentOuter: React.FunctionComponent<ICanvasOuterDefaultProps>
```